docs/OIDC-Activation.md

```diff
@@ -0,0 +1,40 @@
+# OpenID Connect (OIDC) Activation
+
+PowerAuth protocol allows activation using OpenID Connect (OIDC) protocol.
+
+
+## OIDC Activation Configuration
+
+To enable OIDC activation, an entry with the key `oauth2_providers` must exist in the table `pa_application_config`.
+Mind that this table supports encryption.
+
+- `providerId` - (Required) Identification of the configuration record, used as a key for the configuration, known by the mobile application.
```

Comment:
Can se separate the sections into required and optional parameters? It's a bit strange to use the (Required) or (Optional) word each time.

Reply:
A good idea, I will improve that.